src/hooks/useDecodeTx.ts

```diff
@@ -14,7 +14,7 @@ const useDecodeTx = (tx?: SafeTransaction): AsyncResult<DecodedDataResponse> =>
   const nativeTransfer = isEmptyData && !isRejection ? getNativeTransferData(tx?.data) : undefined
 
   const [data = nativeTransfer, error, loading] = useAsync<DecodedDataResponse>(() => {
-    if (!encodedData || isEmptyData) return
+    if (!encodedData || isEmptyData || !tx.data.to) return
```

Comment:
The parameter should be optional now so I think we can remove this check.

Reply:
Adjusted in 12116b71f0067a697e64c861fc682219a9014108.